packages/slate-editor/src/extensions/inline-links/types.ts

```diff
@@ -0,0 +1,6 @@
+export interface InlineLinksExtensionConfiguration {
+    predefinedLinks?: {
```

Comment:
I was thinking about making this into an array of objects but given our current requirements there's no need.

Reply:
Maybe we should call it `predefinedHrefs`? 🤔 Not sure.

Reply:
What is the value that will be passed to this prop from the app?

Reply:
```
predefinedLinks: {
    label: 'Prezly content',
    options: [
        { label: 'Link to unsubscribe', value: '%unsubscribe_url%' },
        { label: 'Link to story', value: '%release.url%' },
        { label: 'Short link to story', value: '%release.shorturl%' },
    ],
}
```

Reply:
Looks fine as is, if you ask me :)